kernel/syno-ipq806x-1.2/digests

```diff
@@ -1,3 +1,3 @@
-synogpl-7742-ipq806x.tbz SHA1 a335c22c8bc67a795181c976b43ffee0ea07317d
-synogpl-7742-ipq806x.tbz SHA256 e7a8bbc3129ce4dbaac5c1c054c710015d33cec448c74209805152554e9f95c3
-synogpl-7742-ipq806x.tbz MD5 c92af5840b1296d6485fddd94ead4308
+ipq806x-linux-4.4.x.tar.xz SHA1 7cad56bfbbd5071f63e671166046ae97b2c87bbc
+ipq806x-linux-4.4.x.tar.xz SHA256 c44f7139bc416f0f8b96879b5718e2d33be96adf7a4cb38ae65b04394298aaf2
+ipq806x-linux-4.4.x.tar.xz MD5 8bf294be92f7d1e3ea04b41d915a08e7
```

Comment:
This mistake should be fixed in a dedicated "urgent" PR. @hgy59 @th0ma7

Reply:
If I remember correctly I extracted the kernel from the huge archive on a VPS and only downloaded the kernel to my machine, so the new digests are probably wrong here too.